Source/control.cpp

```diff
@@ -1788,18 +1788,18 @@ void DrawSpellBook()
 
 	spl = plr[myplr]._pMemSpells | plr[myplr]._pISpells | plr[myplr]._pAblSpells;
 
-	yp = 215;
+	yp = (55 + SCREEN_Y);
 	for (i = 1; i < 8; i++) {
 		sn = SpellPages[sbooktab][i - 1];
 		if (sn != -1 && spl & (__int64)1 << (sn - 1)) {
 			st = GetSBookTrans(sn, TRUE);
 			SetSpellTrans(st);
-			DrawSpellCel(RIGHT_PANEL + 75, yp, pSBkIconCels, SpellITbl[sn], 37);
+			DrawSpellCel(RIGHT_PANEL + (11 + SCREEN_X), yp, pSBkIconCels, SpellITbl[sn], 37);
 			if (sn == plr[myplr]._pRSpell && st == plr[myplr]._pRSplType) {
 				SetSpellTrans(RSPLTYPE_SKILL);
-				DrawSpellCel(RIGHT_PANEL + 75, yp, pSBkIconCels, SPLICONLAST, 37);
+				DrawSpellCel(RIGHT_PANEL + (11 + SCREEN_X), yp, pSBkIconCels, SPLICONLAST, 37);
 			}
-			PrintSBookStr(10, yp - 23, FALSE, spelldata[sn].sNameText, COL_WHITE);
+			PrintSBookStr(-54 + SCREEN_X, yp - 23, FALSE, spelldata[sn].sNameText, COL_WHITE);

```

Comment:
Something about this doesn't quite make sense. Does `PrintSBookStr()` print 64px of nothing first, or is it actually printing to a temp buffer?

Reply:
looks like the correct change is changing adjusting `PrintSBookStr()`:
```
	sx = x + RIGHT_PANEL + 120;
```
->
```
	sx = x + RIGHT_PANEL_X + SPLICONLENGTH;
```

Reply:
And `control_print_talk_msg()` line 2075:
```
	*x += 264;
```
->
```
	*x += 200 + SCREEN_X;
```